recipes/mlflow/meta.yaml

```
@@ -0,0 +1,80 @@
+{% set name = "mlflow" %}
+{% set version = "0.9.0.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 475a860361c972b428d68a9fc6015f41bf4fc3f225c425a2780ff5ac56a53ccd
+
+build:
+  number: 0
+  entry_points:
+    - mlflow=mlflow.cli:cli
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  noarch: python
+
+requirements:
+  host:
+    - python
+    - setuptools
+    - pip
+  run:
+    - boto3 >=1.7.12
+    - click >=7.0
+    - cloudpickle >=0.6.1
+    - databricks-cli >=0.8.0
+    - docker-py >=3.6.0
+    - entrypoints
+    - flask
+    - gitpython >=2.1.0
+    - gunicorn
+    - mleap >=0.8.1
+    - numpy
+    - pandas
+    - pip
+    - protobuf >=3.6.0
+    - python
+    - python-dateutil
+    - pyyaml
+    - querystring_parser
+    - requests >=2.17.3
+    - scikit-learn
+    - scipy
+    - simplejson
+    - six >=1.10.0
+    - sqlparse
+
+test:
+  imports:
+    - mlflow
+    - mlflow.azureml
+    - mlflow.entities
+    - mlflow.models
+    - mlflow.projects
+    - mlflow.protos
+    - mlflow.pyfunc
+    - mlflow.rfunc
+    - mlflow.sagemaker
+    - mlflow.server
+    - mlflow.store
+    - mlflow.tracking
+    - mlflow.utils
+  commands:
+    - mlflow --help
+
+about:
+  home: https://mlflow.org/
+  license: Apache 2.0
+  license_family: APACHE
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

There is no need for the `RECIPE_DIR` here:
```suggestion
  license_file: LICENSE
```
(I'll merge once this is done and the CI passes, ping me if needed)